src/app/code/community/Shopgate/Cloudapi/Helper/Request.php

```diff
@@ -27,6 +27,16 @@ class Shopgate_Cloudapi_Helper_Request extends Mage_Core_Helper_Abstract
      */
     const KEY_SGCLOUD_INAPP = 'sgcloud_inapp';
 
+    /**
+     * Parameter indicating a shopgate cloud is checkout
+     */
+    const KEY_SGCLOUD_IS_CHECKOUT = 'sgcloud_is_checkout';
+
```

Comment:
According to the ticket, the parameter is called 'sgcloud_checkout'.

Reply:
done